openlibrary/templates/lib/nav_foot.html

```diff
@@ -50,7 +50,7 @@ <h2>$:_('Help')</h2>
     </div>
     <hr>
     <div id="footer-details">
-      <img id="archive-logo" src="/static/images/pantheon.png" alt="" aria-hidden="true">
+      <img id="archive-logo" src="/static/images/pantheon.png" alt="Archive Logo" aria-hidden="true">
```

Comment:
Internet Archive Logo or archive.org logo might be more appropriate here. Will someone using a screenreader know what "Archive" means?

Reply:
Would "Internet Archive Logo" work ?

Reply:
Note, best to wait for @bpmcneilly though as this answer may change :)

Reply:
Yess!